monocle/gerrit/review.py

```diff
@@ -130,6 +130,7 @@ def extract_pr_objects(review):
                 'repository_fullname': review['project'],
                 'repository_shortname': "/".join(
                     review['project'].split('/')[1:]),
+                'url': '%s/c/gerrit/+/%s' % (self.base_url, review['_number']),
```

Comment:
This works better:
'url': '%s/%s' % (self.base_url, review['_number']),

Reply:
done